mslice/plotting/plot_window/plot_figure.py

```diff
@@ -49,12 +52,17 @@ def _display_status(self,status):
             self.actionKeep.setChecked(False)
 
     def _plot_options(self):
-        config = self._get_plot_description()
-        new_config = PlotOptionsDialog.get_new_config(config)
+        if self.canvas.figure.gca().get_images():
```

Comment:
It feels as if we need a helper function somewhere such as `is_cut_figure()` and `is_slice_figure()` so that we can encapsulate this check and make the code more readable. What do you think?

We could also use common python approach to reduce the amount of similar code we type:
```
if self.canvas.figure.gca().get_images():
    view_class, presenter_class = SlicePlotOptions, SlicePlotOptionsPresenter
else:
    view_class, presenter_class = CutPlotOptions, CutPlotOptionsPresenter
new_config = presenter_class(view_class(), self).get_new_config()
...
```